libctru/source/applets/browser.c

```diff
@@ -0,0 +1,20 @@
+#include <3ds/types.h>
+#include <3ds/services/apt.h>
+
+#include <3ds/applets/browser.h>
+
+#include <stdlib.h>
+#include <string.h>
+
+void browserOpenUrl(const char* url)
+{
+	size_t url_len = url ? (strlen(url) + 1) : 0;
+	// check that we passed a url, it isn't too long and not too short (needs at least "http://")
+	// if that is not the case, we just claim our buffer was 0 in size
+	if (url_len > 0x400 || url_len < 8) url_len = 0;
+	// The URL needs to be on the heap, else we get an ARM11 exception when resuming the application
```

Comment:
Is this comment right? Isn't it about write permissions not existing, since APT expects an in/out buffer? (and obviously a string literal resides in rodata...)

Reply:
Clarified the comment